.github/workflows/github-actions-docs-test.yml

```diff
@@ -7,28 +7,33 @@ jobs:
   docs-test-job:
     name: 'Test docs for Tcl syntax and README'
     runs-on: ubuntu-latest
+    container:
+      image: openroad/ubuntu-cpp20
 
     steps:
     - name: Checkout repository
       uses: actions/checkout@v3
       with:
-        submodules: true
+        submodules: 'recursive'
 
     - name: Install required package
       run: |
-        sudo apt-get update && sudo apt-get install -y pandoc
-    - name: Set up Python
-      uses: actions/setup-python@v5
-      with:
-        python-version: '3.10'  
+        apt-get update 
+        apt-get install -y pandoc
+
+    - name: Build messages
+      run: |
+        python3 docs/src/test/make_messages.py
 
     - name: Preprocess files
       run: |
-        cd docs && make preprocess -j${nproc}
+        cd docs
+        make preprocess -j${nproc}
     - name: Run Tcl syntax parser
       run: |
         python3 docs/src/test/man_tcl_params.py
     - name: Run readme parser
       run : |
-        cd docs && make clean
+        cd docs
+        make clean
```

Comment:
Above you do `make preprocess` and here `make clean` - is this intentional?

Reply:
Yes, the second one is a README parsing check. So it doesn't require manpages.